ddtrace/ext/git.py

```diff
@@ -13,3 +27,85 @@
 
 # Git Tag
 TAG = "git.tag"
+
+# Git Commit Author Name
+COMMIT_AUTHOR_NAME = "git.commit.author.name"
+
+# Git Commit Author Email
+COMMIT_AUTHOR_EMAIL = "git.commit.author.email"
+
+# Git Commit Author Date (UTC)
+COMMIT_AUTHOR_DATE = "git.commit.author.date"
+
+# Git Commit Committer Name
+COMMIT_COMMITTER_NAME = "git.commit.committer.name"
+
+# Git Commit Commiter Email
+COMMIT_COMMITTER_EMAIL = "git.commit.committer.email"
+
+# Git Commit Committer Date (UTC)
+COMMIT_COMMITTER_DATE = "git.commit.committer.date"
+
+# Git Commit Message
+COMMIT_MESSAGE = "git.commit.message"
+
+log = logging.getLogger(__name__)
```

Comment:
please use the internal `ddtrace.internal.logging.getLogger` instead so we get a `DDLogger`/rate limited logger